packages/documentation/src/stories/foundation/accessibility/forms/forms-reference-labels.sample.html

```diff
@@ -0,0 +1,15 @@
+<form>
+    <div role="group" aria-labelledby="complex-label">
+      <span id="complex-label">Select your favorite fruits:</span>
```

Comment:
Accessibility requires all texts to be inside `<p>` tags: 
```suggestion
      <p id="complex-label">Select your favorite fruits:</p>
```